docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -11,20 +11,58 @@ Ischemic heart disease (IHD) is a non-communicable cardiovascular disease which
 
 Acute coronary events may cause symptoms such as angina, cold sweats, dizziness, nausuea, neckpain, shortness of breath, sleep disturbances, or weakness. 
 
-Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpace. 
+Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpain. 
 
 .. _NIH: https://www.nhlbi.nih.gov/health-topics/ischemic-heart-disease
 
 
-Modeling Ischemic Heart Disease in GBD 2017
+Modeling Ischemic Heart Disease (IHD) in GBD 2017
 -------------------------------------------
 
 Myocardial infarction (MI) in GBD 2017
 ++++++++++++++++++++++++++++++++++++++
 
-
-Modeling Strategy for Ischemic Heart Disease
-++++++++++++++++++++++++++++++++++++++++++++
+GBD 2017 models IHD using Myocardial infarction (MI) sequelae to estimate the prevalence of IHD, due to the challenges of disease detection and varying symptoms across the population.
+
+
+1. Acute myocardial infarction (MI): Definite and possible MI according to the third universal
+definition of myocardial infarction:
+
+a. When there is clinical evidence of myocardial necrosis in a clinical setting consistent with
```

Comment:
These nested lists are not rendering well in GitHub.  Maybe there is some tweak to them that will look better.

Reply:
Thank you for the feedback! I'll find a better method and make an update.

Reply:
@SantoniS I think the problem might be with your indentation and missing blank lines. See these references for the correct syntax:

https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists

Basically, the two rules you need to follow for lists (including nested lists) are:

1. A blank line is required before the first item in the list and after the last item in the list, *including sublists*. For example, your list numbered i. - v. is not a proper sublist because it is not preceded or followed by a blank line.

2. The items in a sublist must be aligned with the *text* in the parent list. For example, the list labeled a. - d. is not a sublist, because the labels a. b. c. d. are not indented 3 spaces to align with the word "Acute" in the parent list's item 1.

Reply:
Thank you for the information! I'll make these updates shortly.